### DISCUSSION TITLE
In table panel allow showing table cell value as tooltip

### DISCUSSION THREAD

**Discussion by pprkut** [grafana/grafana#52540] [Dashboards, Panels & Visualizations]
Grafana versions before 9.0, when having `Cell display mode` set to `JSON View` showed the cell value as a tooltip. Grafana 9.0 introduced `Cell value inspect`, which replaced the tooltip.

Now, while the modal window is generally nicer to look at, and allows more actions associated with the cell data, it's not as easy to work with as the tooltip. Before this, you could hover over a cell and see the value quickly by just moving the cursor. Need to see the value of a different row? Just move the cursor. With this you could very quickly explore your data.
The modal window makes this much more difficult. One click to open the window and see the data, then you need to move the cursor to the `x` to close the window, but that moves you away from the row you were looking at, so you have to find it again to identify the next row you might want to look at. It's not a big issue, but it adds up and slows down data exploration quite a bit.

Now, I'm not saying the modal window is useless and I'd like the old behavior back, because for some situations this seems very useful. But I think it would be nice to have a `Cell value tooltip` kind of toggle, where you can enable it when you want. This could even work additionally to `Cell value inspect`, where you can use the tooltip for a quick peak and open the modal window to look at it in more detail.

**Comment by Ridlsan** [comment]


**Reply by pprkut** [reply]
I did mention that in my initial post. This discussion isn't about `JSON View` not working, but about adding an option like `Cell value inspect` that would enable a tooltip view rather than a modal window.

**Reply by Ridlsan** [reply]
My bad, totaly agree, also width in inpsect is not fixed, so u have to scroll, would be awesome to allow fix it

**Comment by Yui-Song** [comment]
Hello, any updates about this discussion? Is it possible to get the old behavior back?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
